tests/test_variant_patterns.py

```diff
@@ -10,7 +10,7 @@
 from odxtools.odxdoccontext import OdxDocContext
 from odxtools.odxlink import DocType, OdxDocFragment
 
-doc_frags = [OdxDocFragment(doc_name="pytest", doc_type=DocType.CONTAINER)]
+doc_frags = (OdxDocFragment(doc_name="pytest", doc_type=DocType.CONTAINER),)
```

Comment:
this is not an issue introduced by this PR, but I think that in practice all IDs that are inside a container also contain a document fragment for the layer which they are contained in? (this does not really matter here, though...)

Reply:
Attributes of the ODX category like company data could have just the container though, right?

Reply:
right.